spec/acceptance/class_logstash_spec.rb

```diff
@@ -1,40 +1,41 @@
+# frozen_string_literal: true
+
 require 'spec_helper_acceptance'
 
 shared_examples 'a logstash installer' do
-  it "should install logstash version #{LS_VERSION}" do
+  it "installs logstash version #{LS_VERSION}" do
     expect(shell('/usr/share/logstash/bin/logstash --version').stdout).to eq("logstash #{LS_VERSION}\n")
   end
 
   case fact('osfamily')
   when 'RedHat', 'Suse'
     describe package('logstash') do
-      it { should be_installed }
+      it { is_expected.to be_installed }
     end
   when 'Debian'
     # Serverspec has been falsely reporting the package as not installed on
     # Debian 7, so we'll implement our own version of "should be_installed".
-    it "should install logstash package version #{logstash_package_version}" do
+    it "installs logstash package version #{logstash_package_version}" do
       apt_output = shell('apt-cache policy logstash').stdout
       expect(apt_output).to include("Installed: #{logstash_package_version}")
     end
   end
 
   describe service('logstash') do
-    it { should be_running }
-    it 'should be_enabled' do
-      if fact('lsbdistdescription') =~ /centos release 6/i
-        skip('Serverspec seems confused about this on Centos 6.')
-      end
-      should be_enabled
+    it { is_expected.to be_running }
+
+    it 'be_enableds' do
+      skip('Serverspec seems confused about this on Centos 6.') if fact('lsbdistdescription') =~ %r{centos release 6}i
+      is_expected.to be_enabled
     end
```

Comment:
No more need to check for CentOS 6 since you're dropping it.
```suggestion
    it { is_expected.to be_enabled }
```